### DISCUSSION TITLE
Caching ApplicationContext.GetRequiredService<IDataPortal<ChildWidget.Factory>>

### DISCUSSION THREAD

**Discussion by michaelcsikos** [MarimerLLC/csla#3147] [Questions]
In a `BusinessListBase` is it OK to cache the `IDataPortal<ChildWidget.Factory>` instead of creating a new instance every time a child is created?

**Comment by rockfordlhotka** [comment]
I think that should be fine as long as the cache doesn't extend outside the current DI scope. In other words, you can't cache it in a static, but you can probably cache it in a local variable within the data portal operation method.
